src/src/Repository/OptionRepository.php

```diff
@@ -19,6 +19,32 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Option::class);
     }
 
+    public function findDomainName(): ?string
```

Comment:
Util functions for getting at config in a nicer way - will be easier to mock too :+1: